src/test/java/com/google/starfish/services/MiscNoteLabelServiceTest.java

```diff
@@ -0,0 +1,134 @@
+package com.google.starfish.services;
+
+import java.util.Date;
+import java.util.Calendar;
+import java.sql.SQLException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import javax.sql.DataSource;
+import java.util.Arrays;
+import static com.ninja_squad.dbsetup.Operations.*;
+import static org.junit.Assert.assertTrue;
+
+import com.ninja_squad.dbsetup.DbSetup;
+import com.ninja_squad.dbsetup.destination.DataSourceDestination;
+import com.ninja_squad.dbsetup.operation.Operation;
+
+@RunWith(JUnit4.class)
+public class MiscNoteLabelServiceTest {
+  private Constants constants = new Constants();
+  private String LABELS = constants.LABELS;
+  private String NOTES = constants.NOTES;
+  private String MISC_LABELS = constants.MISC_LABELS;
+  private DataSource pool = Constants.pool;
+  private MiscNoteLabelService miscNoteLabelService = new MiscNoteLabelService();
+
+  private final String miscLabelOne = "best";
+  private final String miscLabelTwo = "hard";
+  private final String miscLabelThree = "math";
```

Comment:
(Here and elsewhere) when you have hard-coded values like this, I think it helps readability to make these styled as constants (i.e. all caps) so that we can clearly distinguish in the test when you have a derived value (the thing you're usually testing) vs. an expected value

Reply:
Definitely agree with this, my mistake for not making the constants all caps from the beginning!